### ISSUE TITLE
Capture job structure in Jenkins after adding release, evaluation type of jobs

### ISSUE THREAD

**Issue by zdtsw** [adoptium/ci-jenkins-pipelines#586]
Use this issue to catch what has been changed in the Jenkins job/pipeline regarding nightly, release and evaluation type.

Open discussion:

- if these are the best ways, 
- which part can be improved, 
- it is easy for user to navigate around
- etc

Ref: https://github.com/adoptium/ci-jenkins-pipelines/issues/473

**Comment by zdtsw** [comment]
below jobs should be cleaned up to not make more confusion:

- https://ci.adoptopenjdk.net/job/build-scripts/job/jobs/job/evaluation/job/jobs/job/jdk11u/*
- https://ci.adoptopenjdk.net/job/build-scripts/job/jobs/job/evaluation/job/jobs/job/jdk8u/*-prototype-*
- https://ci.adoptopenjdk.net/job/build-scripts/job/jobs/job/evaluation/job/jobs/job/jdk17u/*-prototype*
- https://ci.adoptopenjdk.net/job/build-scripts/job/jobs/job/evaluation/job/jobs/job/jdk19u/jdk19u-prototype-windows-aarch64-temurin


Take jdk17 as example:
````
└── build-scripts
    ├── jobs
    │   ├── jdk17u
    │   │   ├── jdk17u-linux-x64-temurin            **(called by nightly and weekly)**
    │   │   └── jdk17u-linux-x64-temurin_SmokeTests **(called by nightly and weekly)**
    │   ├── evaluation
    │   │   └── jobs
    │   │       └── jdk17u
    │   │           ├── jdk17u-evaluation-alpine-linux-aarch64-temurin            **(called by nightly evaluation and weekly evaluation**)
    │   │           └── jdk17u-evaluation-alpine-linux-aarch64-temurin_SmokeTests **(called by nightly evaluation and weekly evaluation**)
    │   └── release
    │       └── jdk17u
    │           ├── jdk17u-release-linux-x64-temurin              **(called by release**)
    │           └── jdk17u-release-linux-x64-temurin_SmokeTests   **(called by release**)
    ├── opendjk17-pipeline                          **(nightly pipeline or called by weekly)**                            
    ├── evaluation-openjdk17-pipeline               **(evaluation pipeline run at night or called by weekly evaluation)**
    ├── weekly-openjdk17-pipeline
    ├── weekly-evaluation-openjdk17-pipeline
    ├── release-openjdk17-pipeline                  **(release pipeline called by releaseTrigger job or manually)**
    ├── release
    │   ├── create_installer_*
    │   ├── refactor_openjdk_relase_tool
    │   └── sign_*
    ├── utils
    │   ├── build-pipeline-generator                    **( generate nightly and weekly pipeline)**
    │   ├── evaluation-pipeline-generator               **(generate evaluation and weekly evalution pipeline)**
    │   ├── release-build-pipeline-generator            **(generate release pipeline)**
    │   ├── evaluation-pipeline_jobs_generator_jdk17u   **(generate downstream jdk17  jobs for evaluation pipeline)**
    │   ├── pipeline_jobs_generator_jdk17u              **(generate jdk17 downstream jobs for  nightly pipeline)**
    │   ├── release_pipeline_jobs_generator_jdk17u      **(generate jdk17 downstream jobs for release pipeline)**
    │   ├── releaseTrigger_jdk17u                       **(auto trigger release pipeline in release weeks)**

````
jobs under`build-scripts.jobs.jdk17u` should not run on the same platform*OS as jobs under `build-scripts.jobs.evaluation.jobs.jdk17u`
jobs under `build-scripts.jobs.release.jdk17u` should be contained by the jobs under `build-scripts.jobs.jdk17u`
**In short:   a downstream job  either lives in `build-scripts.jobs.jdk17u` and `build-scripts.jobs.release.jdk17u` or  `build-scripts.jobs.evaluation.jobs.jdk17u`**


At the moment, we have not migrated openj9, bisheng (non-temurin variant) jobs from `build-scripts.jobs.jdk17u`  to `build-scripts.jobs.evaluation.jobs.jdk17u`
But the plan is to do so.
Plus, jobs with name "prototype" should be removed as well.
